roles/contrail-provision-dockers/tasks/main.yaml

```diff
@@ -0,0 +1,29 @@
+- name: Debug hostvars 
```

Comment:
Should it be removed from the job before merging?

Reply:
Removed